test/test_plugin_helper.rb

```diff
@@ -20,3 +20,17 @@ class Spec
     end
   end
 end
+
+module ForemanOmaha
```

Comment:
Does it make sense to have this in core as well?

Reply:
It makes sense, there is a problem with test when we use `webpacked_plugins_js_for` and we try to register fake `:skip_test` plugin. This is to make possible to register `:skip_test` plugin.
I've checked and when https://github.com/theforeman/foreman/blob/develop/test/test_helper.rb#L30-L32 is commented out then the test passes as well.

Reply:
We have https://github.com/theforeman/foreman/blob/1d28598ff33ed287d192754deaee988508e18ea0/app/registries/foreman/plugin/assets.rb#L33-L35 so I wonder if that could be used